Realm/Realm.Weaver/Analytics/Analytics.cs

```diff
@@ -233,19 +233,22 @@ private void AnalyzeUserAssembly(ModuleDefinition module)
                 _realmEnvMetrics[UserEnvironment.UserId] = GetAnonymizedUserId();
                 _realmEnvMetrics[UserEnvironment.ProjectId] = SHA256Hash(Encoding.UTF8.GetBytes(module.Name));
                 _realmEnvMetrics[UserEnvironment.RealmSdk] = ".NET";
+                _realmEnvMetrics[UserEnvironment.RealmSdkVersion] = module.FindReference("Realm")?.Version.ToString();
```

Comment:
The weaver version and the Realm version are the same. So rather than going through FindReference, you can give it the version of the weaver assembly.